src/api/uploads/__tests__/FolderUploadNode.test.js

```diff
@@ -182,6 +184,49 @@ describe('api/uploads/FolderUploadNode', () => {
 
             expect(folderUploadNodeInstance.addFolderToUploadQueue).not.toHaveBeenCalled();
         });
+
```

Comment:
May need a afterEach lifecycle method with restoreAllMocks since we are mocking the sleep method.